### PR TITLE
Standardize stylelint rules

### PR DESCRIPTION
**Pull request by HouCoder**
## Summary of changes

- Add basic Stylelint configurations.

/fyi: @ben-juwai , @CherishG , @AlanWenhao : Please review.

### CODE REVIEW

**Review by CherishG — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        'order/properties-order': [
+            // Positioning
+            'position',
+            'top',
+            'right',
+            'bottom',
+            'left',
+            'z-index',
+
+            // Display & Box-model
```
Comment (AlanWenhao) [inline index.js:59]: Is this mean that we should write css by this order in our next development ?

**Review by AlanWenhao — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (HouCoder) [reply index.js:59]: yep, I think it's necessary for a better organized codebase.

**Comment by HouCoder** [comment]
Thanks guys, closing it.

File: index.js (hunk 1 above)
Comment (AlanWenhao) [reply index.js:59]: BTW,and I remembered that we left a blank line between `layout-css` and `composite-css`,should we still leave it or it dose no matter

Comment (HouCoder) [reply index.js:59]: I tried to have a blank line between each group, unfortunate the linter does not support this feature. 😢

Comment (AlanWenhao) [reply index.js:59]: Oh~~
I got the reason :face_with_head_bandage:

**Pull request merged**